lib/gamnit/android_two_fingers_motion.nit

```diff
@@ -0,0 +1,113 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Two fingers camera manipulation, scroll and pinch to zoom
+#
+# Provides the main service `EulerCamera::accept_two_fingers_motion`.
+module android_two_fingers_motion
+
+# TODO add an abstraction when another platform supports it
```

Comment:
Yeah... But there was too many configuration options for the desktop: we'd have to have a scoll multiplier and choose between `wasd` and the arrow keys. So it was a bit different and also platform specific. It would be worth it, but it requires more thought.

I'm thinking of moving this service to a subclass of the future `Scene`, so you could have something like a `GestureScene` with different platform specific implementations.